### ISSUE TITLE
upgrade lodash from '3.10.1'

### ISSUE THREAD

**Issue by mbana** [Polymer/web-component-tester#342]
hi, this is an old version, pls do upgrade.

**Comment by panuhorsmalahti** [comment]
Yes, I believe lodash should be updated to 4.x. It's now causing bower install conflicts.

**Comment by slhenty** [comment]
+1
The old version of lodash (3.x) doesn't allow mocking the clock with sinon.fakeTimers. Later versions of lodash (4.x) apparently do.

**Comment by rictic** [comment]
The issue here is that it would be a breaking change to all WCT users, and updating lodash is infamously tricky. Honestly, WCT should never have bundled lodash in the first place.

Our preferred solution to this would be to remove WCT's usage of lodash entirely, that way projects can specify their preferred version of lodash in their bower.json files.

**Comment by panuhorsmalahti** [comment]
"updating lodash is infamously tricky"

Really? I've never seen that.

**Comment by rictic** [comment]
Perhaps I've just been unlucky?

I guess I'm also thinking about occasions when I wanted to replace lodash methods with the equivalent ES5 and I've found it difficult to track down how a particular method will handle nulls, undefineds, falsy objects, etc.

In any case, we'd like to avoid breaking users' tests, at least until we move to unbundling the helper libraries like lodash that WCT ships.

**Comment by bennypowers** [comment]
I'm for either solution, but it should either be upgraded to 4 or removed so we can use our own libraries.

**Comment by justinfagnani** [comment]
We're working on a new version of WCT that doesn't include any extra client-side dependencies.

**Comment by Drenmi** [comment]
@justinfagnani: What's the time scale for that? Weeks? Months?

Is there any way we can help out?
